### ISSUE TITLE
typescript error in the build process

### ISSUE THREAD

**Issue by mackovershay** [optimizely/javascript-sdk#608]
After I updated the optimizley-sdk to version 4.4.0 my build process started to fail
I am using typescript(4.0.5) and I started to get the following errors after the upgrade:
node_modules/@optimizely/optimizely-sdk/lib/optimizely/index.ts:150:11 - error TS2322: Type 'UserProfileService' is not assignable to type 'null'.

150           userProfileService = config.userProfileService;
              ~~~~~~~~~~~~~~~~~~

node_modules/@optimizely/optimizely-sdk/lib/optimizely/index.ts:335:7 - error TS2322: Type 'string' is not assignable to type 'null'.

335       variationId = projectConfig.getVariationIdFromExperimentAndVariationKey(configObj, experimentKey, variationKey);
          ~~~~~~~~~~~

node_modules/@optimizely/optimizely-sdk/lib/optimizely/index.ts:336:7 - error TS2322: Type 'string' is not assignable to type 'null'.

336       experimentId = projectConfig.getExperimentId(configObj, experimentKey);
          ~~~~~~~~~~~~
I have downgraded the SDK to version 4.3.4 and everything works as expected.


**`@optimizely/optimizely-sdk` version:** 4.4.0


**`node` version:** 12.17

**`npm` version:** 6.14.4

**Comment by chris-armstrong** [comment]
I can confirm I'm seeing this as well. The odd part is that the SDK is packaged with .d.ts files for the compiled JS, so it's not clear why its being installed with the source .ts files (in my understanding, these should be eliminated via a .npmignore file).

**Comment by gitrkaplan** [comment]
I'm using SDK version `4.2.1` and am also seeing this issue.

**Comment by gitrkaplan** [comment]
@mackovershay Until the issue has been resolved on Optimizely's end, using `4.1.0` resolved my issue.

**Comment by yavorona** [comment]
Thanks @mackovershay @chris-armstrong @gitrkaplan for reporting this issue. We merged  https://github.com/optimizely/javascript-sdk/pull/611 and released it in [v4.4.1](https://github.com/optimizely/javascript-sdk/releases/tag/v4.4.1)
